internal/uvm/security_policy.go

```diff
@@ -42,6 +42,17 @@ func WithSecurityPolicyEnforcer(enforcer string) ConfidentialUVMOpt {
 	}
 }
 
+func base64EncodeContent(filePath string) (string, error) {
+	if filePath == "" {
+		return "", nil
+	}
+	content, err := os.ReadFile(filePath)
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
This could result in an undetected file not found.  Callers may end up troubleshooting why the base64 string is "" instead of troubleshooting that the file doesn't exist.

Reply:
sure, that's fair. although I should probably revert this change, since the approach to fetching the certs have changed and in previous iterations, I was reading it from the host, thus introduced this utility function. At the moment we always set the default reference file, so under non-SNP deployments the file won't exist, I might need to refactor that a bit, which is probably outside of the scope of this PR. I'll add some logging for now and make the refactoring change in a different PR, if you don't mind.